internal/cluster/validator.go

```diff
@@ -116,8 +119,9 @@ func (v *clusterValidator) isMachineCidrEqualsToCalculatedCidr(c *clusterPreproc
 	if swag.BoolValue(c.cluster.UserManagedNetworking) {
 		return ValidationSuccess, "The Cluster Machine CIDR is not required: User Managed Networking"
 	}
-	if c.cluster.APIVip == "" && c.cluster.IngressVip == "" {
-		return ValidationPending, "The Machine Network CIDR, API virtual IP, or Ingress virtual IP is undefined."
+	if c.cluster.APIVip == "" && (c.cluster.APIVips == nil || len(c.cluster.APIVips) == 0) &&
```

Comment:
Length of nil slice is 0, so no need to check nil equality.

Reply:
Done